apps/console/src/features/applications/api/application.ts

```diff
@@ -984,3 +984,83 @@ export const getRequestPathAuthenticators = (): Promise<any> => {
                 error.config);
         });
 };
+
+/**
+ * Function to add/update My Account portal status.
+ *
+ * @param {boolean} status - My Account portal status.
+ *
+ * @return {Promise<any>} Promise of type any.
+ * @throws {IdentityAppsApiException}
+ */
+ export const updateMyAccountStatus = (status: boolean): Promise<any> => {
+
+    const config = `{"name": "status", "attributes": [{"key": "enable", "value": ${ status.toString() }}]}`;
```

Comment:
Should this be a string.
Can't we have an object here?

```js
const config: Record<string, unknown> = {
    name: "status",
    ...
```

Reply:
Addressed with 06eac07bbcb1c736313953e0cc52b62e0ac383a7